bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -2060,67 +2064,68 @@ CGINS-PARADK-20582,,Sensor,0,Photosynthetically Available Radiation: PARAD Serie
+CGINS-PARADM-50220,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50220,,,,,
+CGINS-PARADM-50221,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50221,,,,,
+CGINS-PARADM-50222,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50222,,,,,
+CGINS-PARADM-50223,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50223,,,,,
+CGINS-PARADM-50236,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50236,,,,,
+CGINS-PARADM-50237,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50237,,,,,
+CGINS-PARADM-50238,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50238,,,,,
+CGINS-PARADM-50239,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50239,,,,,
+CGINS-PARADM-50240,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50240,,,,,
+CGINS-PARADM-50241,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50241,,,,,
+CGINS-PARADM-50242,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50242,,,,,
+CGINS-PARADM-50243,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50243,,,,,
+CGINS-PARADM-50244,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50244,,,,,
+CGINS-PARADM-50245,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50245,1.2,,,,
+CGINS-PARADM-50252,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50252,,,,,
+CGINS-PARADM-50277,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50277,,,,,Woods Hole Oceanographic Institution
+CGINS-PARADM-50304,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50304,,,,,

```

Comment:
The cleanup looks ok - w/ removal of 'CGSN Glider Instrument Pool' & 'From Teledyne Webb Research's OOI glider instrument pool' from Comments column

Reply:
I removed those because TWR no longer operates an instrument pool between WHOI and OSU (where instruments might swap between MIOs).  We now keep our own set of instruments.

Reply:
Sounds good thx.

Reply:
That is fine.  We should add in the MIO for these when we can.